src/Asset/EntrypointLookup.php

```diff
@@ -89,7 +116,17 @@ private function getEntriesData(): array
                 throw new \InvalidArgumentException(sprintf('Could not find the entrypoints file from Webpack: the file "%s" does not exist.', $this->entrypointJsonPath));
             }
 
-            $this->entriesData = json_decode(file_get_contents($this->entrypointJsonPath), true);
+            $cache = $this->getConfigCacheFactory()->cache($this->getConfigCacheFileName(), function (ConfigCacheInterface $cache) {
+                $content = sprintf(<<<EOF
+<?php
+return %s;
+EOF
+                , var_export(json_decode(file_get_contents($this->entrypointJsonPath), true), true));
+
+                $cache->write($content);
```

Comment:
Here, I believe the second argument is `Resource` - we need 1 resource that points to the `entrypoints.json` file. This is the key to how cache invalidation works in the dev environment: by passing that here, the cache will know to regenerate if that file is updated.